hadoop-tools/hadoop-aws/src/main/java/org/apache/hadoop/fs/s3a/S3AUtils.java

```diff
@@ -96,6 +96,9 @@
 
 /**
  * Utility methods for S3A code.
+ * Some methods are marked LimitedPrivate since they are being used in an
+ * external project. It would be better to call them rather than copying the
```

Comment:
Remove `It would be better to call them rather than copying the
  whole code in that project.` and